cores/esp32/esp32-hal-spi.h

```diff
@@ -97,6 +97,8 @@ void spiSSSet(spi_t *spi);
 void spiSSClear(spi_t *spi);
 
 void spiWaitReady(spi_t *spi);
+//invert hardware SS
+void spiSSInvertout(spi_t *spi, bool invert);
```

Comment:
I think you can remove the `out` of the function name. This will make it easier to read.